src/translations/en.js

```diff
@@ -2,6 +2,10 @@ export default {
   Alert: {
     close: 'Close',
   },
+  FileInputField: {
```

Comment:
This is not related to this PR, but @mbohal, if I see this and my another suggestion to display number of selected files, I think we need more robust approach for translations. For English it is OK, but I guess there can be languages that can have such language rules for which this approach would not work.

Reply:
I think this can be closed as we don't want to address it in this PR, do we?

Reply:
I think that you are right. We are just in problematic situation, because those translations won't work at Czech launguage. We should at least map this issue in new issue. @mbohal?

Reply:
Lets deal with it in another issue. For now all we need is english.

Reply:
Filed under #611, closing here.